### ISSUE TITLE
Namco Museum (not 50th anniversary) has no BGM

### ISSUE THREAD

**Issue by Darkeh** [emukidid/swiss-gc#331]
The original U.S. released Namco Museum has a very strange behavior with Swiss. When using the compressed nkit.iso variant of the game, it won't get passed the title screen and freeze. Moreover, this behavior happens before displaying the "press start" button. However, if the game is uncompressed in its 1:1 iso format, it will work, but with no background music in the title screen or games. 

Specifications:
Swiss v0.5r813 (latest regarding this post)
Boot device: SD Media Launcher
Game folder location: Root of SD2SP2
SD: SanDisk 128GB Extreme MicroSDXC (SDSQXA1-128G-GN6MA) - Formatted to FAT32 in 32k cluster size.
Cheat codes are disabled as well as streaming audio.

**Comment by Extrems** [comment]
This is an issue with NKit itself. The game also panic in Dolphin with the following:
```
DVDPrepareStreamAsync(): Specified start address (filestart(0x58a8a0) + offset(0x9328000)) is not 32KB aligned in "dvdfs.c" on line 1186.

Address:      Back Chain    LR Save
0x806615a8:   0x80661638    0x804785b8
0x80661638:   0x80661670    0x80483784
0x80661670:   0x80661690    0x804a8d18
0x80661690:   0x806616c0    0x804a94f8
0x806616c0:   0x806616c8    0x804a918c
0x806616c8:   0x806616d0    0x80012ec0
0x806616d0:   0x80661710    0x8000f584
0x80661710:   0x80661748    0x8000e644
0x80661748:   0x80661798    0x8000ab70
0x80661798:   0xffffffff    0x800031fc
```
Streaming audio emulation is needed for streaming audio.

**Comment by Extrems** [comment]
You may convert your .nkit.iso back to .iso, add the following to NKit.dll.config, and convert again to .nkit.iso.
```
  <preserveFstFileAlignment>
    <add key="GNMEAF0000" value="8000" />
  </preserveFstFileAlignment>
```

**Comment by Extrems** [comment]
Fixed in 3a544ad (r851).
